Integrations/integration-MinervaLabsAntiEvasionPlatform.yml

```diff
@@ -0,0 +1,293 @@
V3IngYr3Yh+ajPR/ih9xZZC9Qvuo8FM0j7Hqp/ooPyuwVSORSP5F8UKr8HY7sUYozIhHTAdjbjJHZqXxvpbwO0T4BxDZlgarXqd8A+tjZCHlbhdkfooAACAwKluxyAZc6eOuIjkJucrzycrVIxKvRZtkGbNtkzSF7UzelRsDiSUSdwSO/sKFlq8uIgClN0ZNk0hk0Alq3SWusweQ7GxuWMLICDHSp8/IwgQQyBIOXOmbIlGVdaUci65UAAAWA4uawXo/g03zMlcdFT74gb2EPkbI3YRzRQe2jCIsOzLC+4J7CZmUJwzNavlwA58E9DYc2ue3GtsJ585Pg7b4wMqWkS9DxIPnJ+Q/VAEFEiZlFcxCo4CCOkdk3BaAFAHOydhpUmxjSc9j52RTkDiyJzZpAT7oAwJsgIDfFEk8yzijZw6uOnrE96cdMX1CT2FAGzevMF7HayXR8BR5kD3xkogAOaAPDPtwIJOH1PFhE4i5SO8A91IBEEbI01cxG4y+dLshYvIVlLBhIz350fsJnQBp6fjSk3waxR7LtwCXYE3o3ipQA0CM+KjcCKSUMTo+oa0075F5uaTsNysFiAZs5FEGYmQvZnnQcjg8acYGRjKH6pmxCRMzfvFHGXPNBSRt/QPLoU+iQNwLm8kO01MD1uNr2OJ1pLLMMYEntVyTdzmTjigdNnPkTxpWZl3IB9jvzRbwuoT9fHuyyXXD55OXSkrIlTOEzWcPNQwGEJEcEoYAkQkRyp2CGTxVd7oqANSff3xprZT+aehvwNYidIfLjh6jxoMKKKIdcIQkai5yeTUU5Urq81y+hsCOh482x8F2rpBH+8caSmXsQX234GbLHN+An4D6yPJCbdav0kzooTMzonV+SoRJLlAcSHplvdAxdPvjSm4cdQ+m/BTIdygPsfwYqWlZ8E+XcpQbwYpI7AUE2YL40RcaEmNojmfSd+8MiR0GHx+CwQlo4Yx6SkWkGucfSealNiV1l+mtdIl7340bWAP3Q9hv3Ln3i/Bm+i0ZwMeqluHaaAl6+aKVuuagr4ce4sT5b9t/nc33+FDoms4Z/HujZCwcIQPOBeg4/wDlNb4ssv4VtvYwwSQYelN0coQ5q8fmSPTfMIMn/hGQP4G+AMwH8/8AC/2PDvjkwe+DA3VC8fRx7BoDPUMedJC7qFXachCP2rCkRfXQKTyoydpbcdQsOj+LFme/BOwaaAzgM3gGJaVMUcs2OGNuHsdSyGBDDDlgpuI5tfk0GMZRlSLjRe3tHDfA5XpEn1KaIsG2JJF6fyVFNGQ55UBGaEpowyZJI7NrgsLmYcIuOsSEsBA3pz8GSZA4I5jwOWHhB+t/+GC3734U4jyA2730NTMqdJGIWAbDlNoavqUdOQjIHHKNb0lEVWVc9oJAMiMI0X/UK028PLJMtxvKKPCmiCzzH1U6SMwsgN0zd+ADxIj2Q9xv2KiX5A/e+ccBmJSOzNJozOAGCiWW/qtJIgmYBGHn3x3LIgDFaE2LmMwEvdlpqZDAocaCzpJwDTyjqvwLft0tG7/fO/EeepbP3RJa6ug2ZWultRa9LtgPGGDPWHlx2vsyRa6zv4J2FIJps4Do4GIyOBN/60P637K/q0+t8nEkEaCI+VT5LaM1SGSGDpfGrTyaCUcDo+SSnbu5Lkmolx0eMCtMKXj+JV58mwwA55BF2x/g4Ev/ALif137T2HuP2m+plhTqwxzgNA38BCAlmG9y1NEsZIYEk86FXFqxiTfDk9adDg+FGInEa4SsC2KcwWXsZ7HTUAGKc6ZOOiV2EliCy+DwItcH8778cHFXov733bQIh/BC1GThGlGImpQHDhwAgNlwNTIDwaHs2iEryJhDEThZ7hJYcTkH6zq0+1qs1zWV2Cau1DJZaC3nUpTV2cPhHAN7+I39ts06rwDcSwtxF3XAatGnVgiWTCWkJdtHTa6gEUtGBGJpiOZanDRGe0VgjkmPBAIJJy+P9rz6bXmMLt0gM5JtsOqptLDz/nAdr+I39rZ7FM8A5gY7Cbd6HpNXK4i5GL5hy0UQdXTibjCAkjCTHHbEMVxT0ddR4hvTrorMZAc1/aUgEEcE3iFswS0vzC/+tt4tY5JnF4AWl4tJOAB2k1BqocDpNW+sbPwbgyNpM1B5YSRmRONGm1hI9ic+TmjJqPXxavny54bR5yHC+Mvric+rV4ZOEc1CJJc3YgKn5Nu2A93bgBitpGRpCMiEKTD6q++BHP8ArUb7Bu+YzwWRFL1sFXox2VpNzTE9pfzjV6NjoEJGItGSzy3CceCBYsHeTuaVGnuX/TcCAJVgKNUIO0A+trM9nGSC5KeqTGKFMwZ/GFTQ1D74E2tijmL8Dv2INdImXpHfhXUgAxcg8yCcylKgCrELLTAimoXlZMCCcBHDUdtpyl0P2/ApQrqSjwEJ2gwYm+G5EZOVS4yIlx6UBmyTAwdBrSB7jc4APIQc5h7TfdsQXrKfBwSCygYq2Cr6jeDkH1SDa02EkSsodnWjFfJUUEr0A2gGKcdEPvZKLQ7bcB5DqU1knbkkm5MRWFwZk5xFgOzRaVzMJsTrgfOZQxr5mm/gvu6g/VKCAFGY7zJNiu88AVhXRPhT5txUbBjcDE1Y0vU2DSddr5ansPmUswDEYu/zassiOgj9bMA6fe5fWgWY2J6yd9wdJbQIZhTEnJoTSXJjLJk/zJU3ZuMXkBHqeKGSWdQA3jovZSKgOUuaEUfzYvZ8imDVAITqbfeDIpvT9cL5YEYhIjxRbVs+CCPdKCDEh+uL7KwyMJewXzX8f9VgHeb6r0Aj6rLb0nwUAEFjayeG8gZahLVHAEA5wuQ3YbT102lwlOxIUiTGI6LTlH1FRRSsYFSS7nKgT1x0b0YfejTfJQBEkBEkeDhwjUQQo6Lc7U2VuaeDQ0nZcVyWKL2ebACA8fgNbLr5GKMEYGN50mZBMJgXlS0lW+OXL/iSpHWfAih1okntE84PqrLjXqVcHVBSh16NpAz7Yr8RRMXx/wDhP//Z
```

Comment:
The image has to be a PNG file and up to 3 MB

Reply:
👍